hosting/views.py

```diff
@@ -686,6 +686,7 @@ def get_context_data(self, **kwargs):
                     disk_size=context['vm']['disk_size'],
                     memory=context['vm']['memory']
                 )
+                context['subscription_end_date'] = vm_detail.end_date()
```

Comment:
`end_date` would be a correct term instead of subscription_end_date ?  ;)

Reply:
thought that `end_date` might be ambiguous to whether it is the end date for the `vm` or the `subscription`, hence named it `subscription_end_date`